src/main/java/calculator/view/Input.java

```diff
@@ -0,0 +1,21 @@
+package calculator.view;
+
+
+public class Input {
+
+
+    public static String[] getInput(String inputString) throws IllegalArgumentException {
+        validate(inputString);
+        return splitFromInput(inputString);
+    }
+
+    private static void validate(String inputString) {
+        if(inputString == null || inputString.isEmpty()){
```

Comment:
```suggestion
        if(inputString == null || inputString.isBlank()){
```
` `같은 공백 문자까지 체크할 수 있게 `isBlank`를 사용하는 게 더 좋겠네요